deployment/ccip/changeset/v1_5_1/cs_configure_token_pools.go

```diff
@@ -70,6 +76,8 @@ type TokenPoolConfig struct {
 	Type deployment.ContractType
 	// Version is the version of the token pool.
 	Version semver.Version
+	// OverrideTokenSymbol is the token symbol to use for the token pool (it is mainly used for clCCIP-LnM settings)
```

Comment:
Let's add a warning here, to only use in scenarios where the token symbol on a particular chain differs from the main tokenSymbol